README.md

```diff
@@ -35,3 +35,9 @@ That work was presented [here](https://www.youtube.com/watch?v=RS1aiQqgUTA).
 Questions? Comments? Ideas?
 Come discuss `ko` with us in the `#ko-build` channel on the [Kubernetes Slack](https://slack.k8s.io)!
 See you there!
+
+### Community Meetings
+
+We have a bi-weekly community meeting on [Thursdays @ 6:00pm UTC time](https://dateful.com/eventlink/2793571750). The main goal of these meetings is that we want to hear from you! We want to know what you're using `ko` for, what you'd like to see in `ko`, how we can make `ko` better for you and with any remaining time we can go through open issues/PRs.
+
+We have a [meeting agenda](https://docs.google.com/document/d/1eQ67Qxwf1tkTv0yU_dw9bIRnlwJZz-5GXCRVOhqbgvU/edit) you can use to propose topics for discussion/ideas. You can also just show up and we'll figure out what to talk about.
```

Comment:
Can we add a short link for the agenda, so `https://ko.build/agenda` redirects to the doc?

That's configured here: https://github.com/ko-build/ko/blob/main/mkdocs.yml#L46

We could also add https://ko.build/meet to redirect to the meet link: https://meet.google.com/xvn-dzzk-wur

Reply:
That makes a lot of sense, cool!